ApiBundle/Tests/Transformer/ContentAttributeTransformerTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Tests\Transformer;
+
+use OpenOrchestra\ApiBundle\Transformer\ContentAttributeTransformer;
+use Phake;
+
+/**
+ * Class ContentAttributeTransformerTest
+ */
+class ContentAttributeTransformerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var ContentAttributeTransformer
+     */
+    protected $transformer;
+
+    /**
+     * set Up
+     */
+    public function setUp()
+    {
+        $this->transformer = new ContentAttributeTransformer();
+    }
+
+    /**
+     * test transform
+     */
+    public function testTransform()
+    {
+        $contentAttribute = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentAttributeInterface');
+        $contentAttributeString = 'ContentAttributeString';
+        Phake::when($contentAttribute)->getName()->thenReturn($contentAttributeString);
+        Phake::when($contentAttribute)->getValue()->thenReturn($contentAttributeString);
+        Phake::when($contentAttribute)->getStringValue()->thenReturn($contentAttributeString);
```

Comment:
You should return 3 different values instead of an unique one. That way you can check the value are correctly setted in the transformer. By returning the same value, the test can't see an inversion between $facade->name, $facade->value or $facade->stringValue;  

By returning the same value for this 3 attributes you can't check that an inversion has been made in the  You should set 3 different values to check that